tests/integration/distribution/keeper/msg_server_test.go

```diff
@@ -206,3 +210,56 @@ func (s *KeeperTestSuite) TestCommunityPoolSpend() {
 		})
 	}
 }
+
+func (s *KeeperTestSuite) TestMsgDepositValidatorRewardsPool() {
+	tstaking := stakingtestutil.NewHelper(s.T(), s.ctx, s.stakingKeeper)
+	tstaking.Commission = stakingtypes.NewCommissionRates(sdk.NewDecWithPrec(5, 1), sdk.NewDecWithPrec(5, 1), math.LegacyNewDec(0))
+	tstaking.CreateValidator(s.valAddrs[1], valConsPk0, sdk.NewInt(100), true)
+
+	testCases := []struct {
+		name      string
+		input     *types.MsgDepositValidatorRewardsPool
+		expErr    bool
+		expErrMsg string
+	}{
+		{
+			name: "happy path",
```

Comment:
Does it work with other denom? If so, can we add a quick case for it?

Reply:
Pushed a non-staking token case.